ecphp/api-gw-authentication-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "EcPhp\\ApiGwAuthenticationBundle\\ApiGwAuthenticationBundle": ["all"]
```

Comment:
I see that you enable the bundle for all environments but the config is only for `dev`, is this really correct?

Reply:
Yes definitely correct.

The use should be able to use some routes only on the `dev` environment.

This is why it is like that.

Reply:
Thank you. Im fine with that. 

Im sure everything is correct. But could you help me understand why the routes is needed only in `dev` but the bundle needs to be active in prod without routes?

Reply:
Sure I can explain.

The package is about JWT authentication. We have to authenticate requests coming from a corporate European Commission service. However, some developers doesn't have the possibility to generate tokens, it can only be generate by that service.

For example, myself. When I developed the bundle for my colleagues, I had no "easy" way to generate a real token for testing.
Also, my colleagues needs to generate fake tokens so they can play with their api and their tests.

This is why that routes exists on `dev` environment only. To be able to generate fake tokens and introspect them.

There is also a note about that in the [documentation](https://ecphp-api-gw-authentication-bundle.readthedocs.io/en/latest/pages/installation.html#step-2).

Reply:
That said, now that I think of it.
As we obviously do not provide the secret key, even if those routes are enabled on `prod`, it would not be an issue.

Reply:
Cool, then I understand. 

I assume that there are some commands too for generating tokens for tests. 

Im happy to merge this PR if you are ready.

Reply:
Thanks, you can merge it safely :-)